src/locales/en/translation.json

```diff
@@ -960,6 +960,42 @@
           "errorRadioMessage": "Select what you would like to do"
         }
       }
+    },
+    "pageUpdateName": {
+      "title": "Update your name using the GOV.UK ID Check app",
+      "header": "Update your name using the GOV.UK ID Check app",
+      "content": {
+        "paragraph1": "To update your details, you need to prove your identity again with your new details.",
+        "paragraph2": "You'll need:",
+        "list1": "photo ID with the name you want GOV.UK One Login to use",
+        "list2": "to tell us where you've lived for the last 3 months",
+        "paragraph3": "Find out <a class=\"govuk-link\" target=\"_blank\" rel=\"noopener noreferrer\" href=\"https://www.gov.uk/using-your-gov-uk-one-login/proving-your-identity\">what photo ID you can use to prove your identity (opens in new tab)</a>.",
+        "warningText": "You may not be able to use your GOV.UK One Login if we cannot confirm your identity using your new details.",
+        "warningTextFallback": "Warning",
+        "subHeading1": "If there is a mistake in your name",
+        "paragraph4": "If there is a mistake in your name. do not try to prove your identity again unless the GOV.UK One Login support team has told you to.",
+        "paragraph5": "For example, if your name is missing letters or a hyphen <a class=\"govuk-link\" target=\"_blank\" rel=\"noopener noreferrer\" href=\"{{url}}\">contact the GOV.UK One Login team (opens in new tab)</a> instead.",
+        "subHeading2": "If you need to use the service now",
+        "paragraph6": "Your proof of identity is still valid even if your details are out of date.",
+        "paragraph7": "You can use your saved details to access the service you need to use.",
+        "subHeading3": "What would you like to do?",
+        "formRadioButtons": {
+          "updateDetailsButtonText": "Update your details using the GOV.UK ID Check app",
+          "updateDetailsButtonTextHint": "We'll guide your through this",
```

Comment:
`"you"` instead of `"your"`

Reply:
thanks - I guess the copy/paste from an image isn't very accurate!